qcodes/tests/test_parameter.py

```diff
@@ -904,6 +905,57 @@ def tearDown(self):
         del self.d
 
 
+class TestParameterScaler(TestCase):
+    def setUp(self):
```

Comment:
Could you add tests for name, label, and unit logic? Also test for metadata working correctly? Also for the fact that an exception is thrown if one is creating a scaler parameter and supplies both gain and decision to the constructor?